### PR TITLE
PLB-97 Error messages and labels for radios 

### PR DESCRIPTION
**Pull request by juff**

### CODE REVIEW

File: src/components/Checkbox/styles.ts [hunk 1]
```diff
@@ -59,6 +60,10 @@ export const StyledLabel = styled.label<StyledLabelProps>`
     `solid ${theme.utilities.inputBorderWidth} ${
       checked && !theme.utilities.useDefaultFocusRect ? theme.colors.primary : theme.colors.inputBorder
     }`};
+  ${({ hasError }) => hasError && css`
```
Comment (Develliot) [inline src/components/Checkbox/styles.ts:63]: I don't think you need the & css

Comment (juff) [reply src/components/Checkbox/styles.ts]: Have pushed an update

File: src/components/LegendWrapper/LegendWrapper.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  legend: 'Legend Example',
+};
+
+const TemplateWithRadio: Story<LegendWrapperProps> = args => (
+  <LegendWrapper {...args}>
+    <Box marginBottom="none">
+      <Radio name="example1" value="one">
+        Option one
+      </Radio>
+    </Box>
```
Comment (Develliot) [inline src/components/LegendWrapper/LegendWrapper.stories.tsx:30]: mutliple radios and checkboxes in the examples would show wether or not we got the spacing correct

File: src/components/LegendWrapper/README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+</>
+```
+
+
+## Props
+
+
+| Name             | Type                   | Required | Default | Description                                                                                                         |
+| :------------ | :------------------- | :-----: | :-------- | :---------------------------------------------------------------------------------------------------------------- |
+| Legend            | string                 | Yes      |         | Fieldset with legend
```
Comment (Develliot) [inline src/components/LegendWrapper/README.md:94]: legend lower case

File: src/components/LegendWrapper/index.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+import defaultTheme from 'src/themes/cruk';
+import ErrorText from 'src/components/ErrorText';
+import Text from 'src/components/Text';
+
+import { StyledFieldset } from './styles';
+
+export type LegendWrapperProps = {
+  children?: ReactNode;
+  /** legend text */
+  legend: string;
```
Comment (Develliot) [inline src/components/LegendWrapper/index.tsx:13]: legendTitle / legend/Text may be

File: src/components/LegendWrapper/index.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  errorMessage?: string;
+  /** flag for error styling */
+  hasError?: boolean;
+  /** when required (required) appears in label */
+  required?: boolean;
+  /** hint text */
+  hintText?: ReactNode | string;
+};
+
+export const LegendWrapper: FC<LegendWrapperProps> = ({
```
Comment (Develliot) [inline src/components/LegendWrapper/index.tsx:24]: `/** documentation about component  */ `using JS docs syntax appears on docs tab of story

File: src/components/LegendWrapper/test.cypress.tsx [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+          <LegendWrapper legend="Legend Example" hasError errorMessage="Error Message">
+            <Checkbox disabled={false} value="value" />
+          </LegendWrapper>
+        </Box>
+      </section>
+    </>
+  );
+};
+
+describe('TextField', () => {
```
Comment (Develliot) [inline src/components/LegendWrapper/test.cypress.tsx:78]: change to Legend wrapper

Comment (Develliot) [reply src/components/LegendWrapper/test.cypress.tsx:78]: delete cyp[ress snap with this label cypress/snapshots/LegendWrapper/test.cypress.tsx/TextField -- should match snapshot.snap.png

File: src/components/Radio/styles.ts [hunk 7]
```diff
@@ -48,16 +48,21 @@ export const Check = styled.span`
...
 
 export const StyledLabel = styled.label<StyledLabelProps>`
   width: 100%;
   position: relative;
   border-radius: ${({ theme }) => theme.utilities.borderRadius};
-  border: ${({ theme, checked }) =>
-    `solid ${theme.utilities.inputBorderWidth} ${
-      checked && !theme.utilities.useDefaultFocusRect ? theme.colors.primary : theme.colors.inputBorder
-    }`};
+
```
Comment (Develliot) [inline src/components/Radio/styles.ts:58]: possibly split out thickness and colour, have key as string and value as string function

File: src/components/index.ts [hunk 8]
```diff
@@ -1,3 +1,4 @@
+import { LegendWrapper } from './LegendWrapper/index';
```
Comment (Develliot) [inline src/components/index.ts:1]: removed

**Review by Develliot — COMMENTED** [review]
Some comments good work

**Comment by juff** [comment]
@Develliot have pushed an update which includes CHANGELOG.md hopefully I have set it out correctly and updated version number in the package.json

File: package.json [hunk 9]
```diff
@@ -32,8 +32,8 @@
     "release": "node release.js",
     "dev": "NODE_ENV=development npm i && docker-compose up server",
     "rollup:build-lib": "npm run clean-lib && rollup --config",
-    "prettier": "prettier --write 'src/**/*.{ts,tsx}'",
-    "prettier:test": "prettier --check 'src/**/*.{ts,tsx}'",
+    "prettier": "prettier --write src/**/*.{ts,tsx}",
```
Comment (Develliot) [inline package.json:35]: does this still work?

**Review by Develliot — APPROVED** [review]
LGTM

**Pull request merged**